module/VuFind/src/VuFind/RecordDriver/SolrDefault.php

```diff
@@ -298,4 +302,74 @@ public function getContainerRecordID()
             && !empty($this->fields['hierarchy_parent_id'])
             ? $this->fields['hierarchy_parent_id'][0] : '';
     }
+
+    /**
+     * Get work identification keys
+     *
+     * @return array
+     */
+    public function getWorkKeys()
+    {
+        return $this->fields['work_keys_str_mv'] ?? [];
+    }
+
+    /**
+     * Return count of other versions available
+     *
+     * @return int
+     */
+    public function getOtherVersionCount()
+    {
+        if (null === $this->searchService) {
+            return false;
+        }
+
+        if (!($workKeys = $this->getWorkKeys())) {
+            return false;
+        }
+
+        if (!isset($this->otherVersionsCount)) {
```

Comment:
This property should be defined in the class with a default of null (unless it's defined somewhere else that I'm missing).

Reply:
Added.